Process.c

```diff
@@ -894,6 +895,48 @@ void Process_writeField(const Process* this, RichString* str, ProcessField field
          xSnprintf(buffer, n, "%3ld ", this->priority);
       break;
    case PROCESSOR: xSnprintf(buffer, n, "%3d ", Settings_cpuId(this->settings, this->processor)); break;
+   case SCHEDPOL:
+      if (this->scheduling_policy < 0) {
+         xSnprintf(buffer, n, "N/A ");
+      } else {
+         #ifdef SCHED_RESET_ON_FORK
+         int pure = this->scheduling_policy & ~SCHED_RESET_ON_FORK;
+         #else
+         int pure = this->scheduling_policy;
+         #endif
+
+         switch (pure) {
+            case SCHED_OTHER:
+               xSnprintf(buffer, n, "OTH ");
+               break;
+            case SCHED_FIFO:
+               xSnprintf(buffer, n, "FF  ");
+               break;
+            case SCHED_RR:
+               xSnprintf(buffer, n, "RR  ");
+               break;
+            #ifdef SCHED_BATCH
+            case SCHED_BATCH:
+               xSnprintf(buffer, n, "BAT ");
+               break;
+            #endif
+            #ifdef SCHED_BATCH
+            case SCHED_IDLE:
+               xSnprintf(buffer, n, "IDL ");
+               break;
+            #endif
+            #ifdef SCHED_DEADLINE
+            case SCHED_DEADLINE:
+               xSnprintf(buffer, n, "DDL ");
+               break;
+            #endif
+            default:
+               CRT_debug("Unexpected scheduling class: %d", this->scheduling_policy);
+               xSnprintf(buffer, n, "??? ");
+               break;
+         }
```

Comment:
I think as most of the entries can be reasonably well described with 4 or 5 characters, I'd suggest resizing this column accordingly, as I'd find the following mapping more descriptive:

```
SCHED_OTHER    : OTH OTHER
SCHED_FIFO     : FF  FIFO
SCHED_RR       : RR  RR
SCHED_BATCH    : BAT BATCH
SCHED_IDLE     : IDL IDLE
SCHED_DEADLINE : DDL EDF
default        : ??? ???
```

EDF AKA Earliest Deadline First, which is (+CBS) what Linux uses in this case.